scss/mixins.scss

```diff
@@ -16,10 +16,6 @@ $_o-brand-define-readme: "https://github.com/Financial-Times/o-brand/#obranddefi
     $validated-brand-name: _oBrandValidateName('Brand', $brand, 'Cannot define brand "#{$brand}" for component "#{$component}"');
     // Validate component name.
     $validated-component-name: _oBrandValidateName('Component', $component, 'Cannot define brand "#{$brand}" for component "#{$component}"');
-    // Do not allow a component brand to be overridden.
-    @if _oBrandIsDefined($component, $brand) {
-        @error 'Brand #{$brand} is already defined for #{$component}.';
```

Comment:
If the `component` and `brand` are the same but the `config` is not the same, we should return an error or a warning informing them about the potential for bugs in their css.